packages/components/src/button/style.scss

```diff
@@ -118,21 +118,14 @@
 		outline: 1px solid transparent;
 
 		&:active:not(:disabled) {
-			background: $components-color-gray-300;
-			color: $components-color-accent-darker-10;
 			box-shadow: none;
 		}
 
-		&:hover:not(:disabled) {
-			color: $components-color-accent-darker-10;
-		}
-
 		&:disabled,
 		&[aria-disabled="true"],
 		&[aria-disabled="true"]:hover {
-			// TODO: Prepare for theming (https://github.com/WordPress/gutenberg/pull/45466/files#r1030872724)
```

Comment:
I think that this comment still applies, right? We should think about adding variables at the theme level for disabled UI

Reply:
The TODO comments were specifically for the issue of `darken`/`lighten` Sass functions being used on theme variables.

As for the [theme-ready gray variables](https://github.com/WordPress/gutenberg/blob/99c3dda88168b262a65ceb33cd6dbce6c5e02b40/packages/components/src/utils/theme-variables.scss#L19-L25), we simply haven't put them to use anywhere yet. So I was kind of thinking of that as a big separate task rather than a contextual TODO item 😄